src/locale/locales/es/messages.po

```diff
@@ -8751,30 +8762,30 @@ msgstr ""
 #: src/components/TagMenu/index.tsx:264
 #: src/view/screens/ProfileList.tsx:701
 msgid "Unmute"
-msgstr "Demutear"
+msgstr "Dejar de silenciar"
 
 #: src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx:155
 #: src/view/com/util/post-embeds/VideoEmbedInner/web-controls/VolumeControl.tsx:94
 msgctxt "video"
 msgid "Unmute"
-msgstr "Desmutear"
+msgstr "Dejar de silenciar"
 
 #: src/components/TagMenu/index.web.tsx:115
 msgid "Unmute {truncatedTag}"
-msgstr "Demutear {truncatedTag}"
+msgstr "Dejar de silenciar {truncatedTag}"
 
 #: src/view/com/profile/ProfileMenu.tsx:258
 #: src/view/com/profile/ProfileMenu.tsx:264
 msgid "Unmute Account"
-msgstr "Demutear Cuenta"
+msgstr "Dejar de silenciar Cuenta"
```

Comment:
Shouldn't the "C" in "Cuenta" be lowercase? I'm not sure since it's capitalized in English too, but in Spanish, it doesn't feel like a proper noun. :D

Reply:
I found many strings capitalized in English and not in Spanish. So for this one, I just modified the translation and not capitalization. But I agree it is more natural in lowercase!